pennylane/devices/default_qubit.py

```diff
@@ -590,6 +624,8 @@ def _create_basis_state(self, index):
         Returns:
             array[complex]: complex array of shape ``[2]*self.num_wires``
             representing the statevector of the basis state
+
+        Note: This function does not support broadcasted inputs yet.
```

Comment:
Only a note as this is a private method.
I don't expect broadcasting support to be particularly important here, but it could be implemented if required.